lib/styles/v2/components/_tag.scss

```
@@ -0,0 +1,43 @@
+$solid-indigo: #5E5CE6;
+$solid-indigo-bg: #EAF3FC;
+
+.nui-tag {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  color: theme("colors.gray.800");
+  line-height: 22px;
+  font-size: 12px;
+  padding: 0 12px;
+  height: 22px;
+  background: white;
+  border: 1px solid theme("colors.gray.400");
+  border-radius: 3px;
+
+  &.nui-tag--large {
+    height: 26px;
+    font-weight: 400;
+    font-size: 14px;
+  }
+
+  &.nui-tag--solid {
+    color: $solid-indigo;
+    background: $solid-indigo-bg;
+    border: 1px solid $solid-indigo;
+  }
+}
+
+.nui-tag--color {
```

```suggestion
.nui-tag__incidator {
```
In BEM, we use underscore for elements and hyphen for variants